frontend/packages/knative-plugin/src/topology/components/nodes/KafkaNode.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
```

Comment:
Definietly. Other component was updated to use different styles.
We also have drop target defined for SBI

I would see this PR still adding targets for Kafka but in generic way.
Since both plugins might be moving to dynamic plugins at some time I think it is good to apply pattern 
of each plugin having shared API that can be published to NPM.

This way knative plugin can use and import shared API even after we move outside console mono repo.
In this situation we can even benefit from it as we could publish our plugin as package and it can be just imported in this PR. WDYT?